setup.py

```diff
@@ -54,7 +54,7 @@
         # noinspection PyUnresolvedReferences
         import dpcppcompiler
         sys.modules["distutils.dpcppcompiler"] = sys.modules["dpcppcompiler"]
-        distutils.ccompiler.compiler_class["dpcpp"] = (
+        distutils.ccompiler.compiler_class["clang-cl"] = (
             "dpcppcompiler", "DPCPPCompiler", "Support of DPCPP compiler"
```

Comment:
May be we should rename DPCPP to clang in compiler definition as well?

Reply:
I don't think it is necessary, because we propose this changes to be able to build dpcpp functionality as well.